DOCUMENTATION.md

```diff
@@ -206,7 +206,8 @@ In the [docker-compose.yml](bookshop-deployment/docker/docker-compose.yml) we ad
 ```
 - JAVA_TOOL_OPTIONS=-javaagent:/workspace/BOOT-INF/lib/opentelemetry-javaagent-1.32.0.jar
 - OTEL_SERVICE_NAME=catalog-service
-- OTEL_EXPORTER_OTLP_ENDPOINT=http://tempo:4317
+- OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4318
```

Comment:
should be jaeger-collector because the jaeger instance in jaeger.yml is called jaeger

Reply:
also port 4317 should be used as we use the Jaeger gRPC interface